src/windows.rs

```diff
@@ -0,0 +1,39 @@
+extern crate winapi;
```

Comment:
Please move to this to the crate root. All `extern crate` statements should be in the crate root.